tools/ci_build/github/azure-pipelines/templates/stages/mac-ios-packaging-build-stage.yml

```diff
@@ -38,7 +38,7 @@ stages:
         cPodName: onnxruntime-training-c
         objcPodName: onnxruntime-training-objc
 
-    timeoutInMinutes: 120
+    timeoutInMinutes: 240
```

Comment:
the packaging pipeline already takes a long time. we should look into splitting per-platform/arch builds into separate jobs that may run in parallel (onnxruntime-extensions does this for its nuget package). can do that in a later PR.

Reply:
yeah. I agree. put 240 minutes here is just for testing. from the observation 180 minutes should be generally enough. 

will look into splitting into different stages.